cluster/manifests/01-coredns-local/daemonset-coredns.yaml

```diff
@@ -58,7 +66,11 @@ spec:
             command:
             - dig
             - "+short"
+{{- if and (eq .Cluster.Provider "zalando-eks") (eq .Cluster.ConfigItems.eks_ip_family "ipv4") }}
             - "@127.0.0.1"
+{{- else }}
+            - "::1"
```

Comment:
Looks like we enter this branch for legacy clusters (IPv4) as well. Does this make sense?

Reply:
And should it be `@::1`?

Reply:
This is unbound which is not yet being activated in EKS clusters. Let's defer fixing this to later.